crates/noirc_evaluator/src/ssa/builtin.rs

```diff
@@ -37,34 +46,42 @@ impl Opcode {
         match self {
             Opcode::LowLevel(op) => {
                 match op {
+                    // Pointers do not overflow
                     BlackBoxFunc::SHA256
                     | BlackBoxFunc::Blake2s
                     | BlackBoxFunc::Pedersen
-                    | BlackBoxFunc::FixedBaseScalarMul => BigUint::zero(), //pointers do not overflow
+                    | BlackBoxFunc::FixedBaseScalarMul => BigUint::zero(),
+                    // Verify returns zero or one
                     BlackBoxFunc::SchnorrVerify
                     | BlackBoxFunc::EcdsaSecp256k1
-                    | BlackBoxFunc::MerkleMembership => BigUint::one(), //verify returns 0 or 1
+                    | BlackBoxFunc::MerkleMembership => BigUint::one(),
                     BlackBoxFunc::HashToField128Security => ObjectType::NativeField.max_size(),
-                    _ => todo!("max value must be implemented for opcode {} ", op),
+                    BlackBoxFunc::RANGE
+                    | BlackBoxFunc::AES
+                    | BlackBoxFunc::AND
+                    | BlackBoxFunc::XOR => {
+                        unreachable!("ICE: max value should be implemented for opcode {} ", op)
+                    }
                 }
             }
             Opcode::ToBits | Opcode::ToRadix => BigUint::zero(), //pointers do not overflow
         }
     }
 
-    //Returns the number of elements and their type, of the output result corresponding to the OPCODE function.
+    /// Returns the number of elements that the `Opcode` should return
+    /// and the type.
     pub fn get_result_type(&self) -> (u32, ObjectType) {
         match self {
             Opcode::LowLevel(op) => {
                 match op {
-                    BlackBoxFunc::AES => (0, ObjectType::NotAnObject), //Not implemented
-                    BlackBoxFunc::SHA256 => (32, ObjectType::Unsigned(8)),
-                    BlackBoxFunc::Blake2s => (32, ObjectType::Unsigned(8)),
+                    BlackBoxFunc::AES => todo!("ICE: AES is unimplemented"),
```

Comment:
Behaviour change here: If/when AES is added into the noir stdlib, we want it to panic the compiler in the places that need to change in CI